Files/MultilingualResources/Files.fr-FR.xlf

```diff
@@ -2015,6 +2035,14 @@
           <source>Disconnect</source>
           <target state="translated">Déconnecter</target>
         </trans-unit>
+        <trans-unit id="PreemptiveCacheParallelLimit.Header" translate="yes" xml:space="preserve">
```

Comment:
I think something went wrong, this string was removed long ago

Reply:
mmmh ? im going to copy the entire actual french manually from the repo...